microsoft-edge/progressive-web-apps-chromium/how-to/icon-theme-color.md

```diff
@@ -60,10 +60,219 @@ Each icon in the `icons` array should come with at least the `src` and `sizes` p
 
 Learn more about [the icons member](https://developer.mozilla.org/docs/Web/Manifest/icons).
 
-<!-- TODO: add information about which sizes Windows require at a minimum, and which sizes are used where -->
+## Icon Image Sizes
```

Comment:
No need for capital letters on each word. So, this should be `Icon image sizes`. See the writing guidelines about capitalization here: https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=33685

Reply:
fixed in the edit.

Reply:
Thanks. I forgot to mention earlier that the 3 other headers also have to be changed in the same way: "Image Level Summary" and "Image Descriptions" and "Target Sized Images".

Reply:
fixed.